sys-apps/systemd/systemd-245.ebuild

```diff
@@ -201,6 +201,7 @@ src_prepare() {
 			"${FILESDIR}/gentoo-Dont-enable-audit-by-default.patch"
 			"${FILESDIR}/gentoo-systemd-user-pam.patch"
 			"${FILESDIR}/gentoo-generator-path-r1.patch"
+			"${FILESDIR}/gentoo-systemd-gnu11.patch"
```

Comment:
Please move this out of the `! use vanilla` condition. There's a line marked `# Add local patches here`; please move this patch to that location.